scripts/run_tests.sh

```diff
@@ -4,7 +4,7 @@ set -e
 BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
 cd "$BASE_DIR"
 
-FAIL_THRESH=40.0
+FAIL_THRESH=100.0
```

Comment:
⁣🎱➖👊➖🎱
  😎👃
⁣💪🎽
  👖
  👟👟


(This is epic, such a flex, and will likely drive me insane in all future contributions)